src/plugins/foxPage/components/OtherOpportunities/OtherOpportunities.tsx

```diff
@@ -0,0 +1,58 @@
+import { Flex } from '@chakra-ui/layout'
+import { Accordion, Text as CText } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text/Text'
+
+import { FoxOpportunities } from '../../FoxCommon'
+import { FoxOtherOpportunityPanel } from './FoxOtherOpportunityPanel'
+
+type OtherOpportunitiesProps = {
+  description: string
+  opportunities: FoxOpportunities
+}
+
+export const OtherOpportunities: React.FC<OtherOpportunitiesProps> = ({
+  description,
+  opportunities,
+}) => {
+  const translate = useTranslate()
+  const hasLiquidityPools = opportunities.liquidityPools?.length
+  const hasFarming = opportunities.farming?.length
+  const hasBorrowingAndLending = opportunities.borrowingAndLending?.length
```

Comment:
Since you're using chunks of sorts, no need for checking truthiness here, you can simply map over them

Reply:
but we need to check the length before mapping, in order to hide the panel, you would prefer to use the length directly as a condition? Thought it was cleaner to name it

Reply:
Yeah, you can early return from the map in that case, in which case react will not render anything (all `falsy` values are effectively not rendering anything)

Reply:
Nop, it's not only about hiding opportunity rows, it's about hiding the whole panel which isn't inside the map at all, it won't work

Reply:
Which is precisely why you'll need proper chunks, e.g a tuple containing the three opportunities. You won't be mapping over individual opportunities, but over chunks, so you can check for the length, or lack thereof of each chunk